src/main/java/frc/robot/commands/autonomousCommands/AutonPathCommand.java

```diff
@@ -0,0 +1,76 @@
+package frc.robot.commands.autonomousCommands;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.RamseteController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.math.trajectory.TrajectoryConfig;
+import edu.wpi.first.math.trajectory.TrajectoryUtil;
+import edu.wpi.first.math.trajectory.constraint.DifferentialDriveVoltageConstraint;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Filesystem;
+import edu.wpi.first.wpilibj2.command.RamseteCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.constants.DriveConstants;
+import frc.robot.subsystems.drive.Drive;
+
+public class AutonPathCommand extends SequentialCommandGroup {
+
+private Drive drive; 
+private DifferentialDriveVoltageConstraint autoVoltageConstraint; 
+private TrajectoryConfig config;
+private RamseteCommand ramseteCommand;
+
+//values to rememeber for robot velocity= 3.66 - Acceleration= 1.83 (these are starting values that was changed over time)
+public final double kMaxSpeedMetersPerSecond = 0.90;
+public final double kMaxAccelerationMetersPerSecondSquared = 0.40;
+public final double MAX_VOLTAGE = 10;
+public Trajectory trajectory = new Trajectory();
+
+    public AutonPathCommand(Drive drive, String trajectoryJSON) {
```

Comment:
Yo, this is a great way of doing this, to pull in the JSON via the constructor so that the command can be agnostic. Well done